decidim-via_oberta_authorization/app/services/via_oberta_authorization_config.rb

```diff
@@ -14,7 +14,7 @@ def url
     end
 
     def purpose
-      if Rails.env == :production
+      if Rails.env == "production"
```

Comment:
Can you try this please?
```suggestion
      if Rails.env.production?
```